workflows/mobilomeannotation.nf

```diff
@@ -142,9 +142,9 @@ workflow MOBILOMEANNOTATION {
     )
     ch_versions = ch_versions.mix(ICEFINDER2_LITE.out.versions)
 
-    genomad_db = Channel.of(file(params.genomad_db, checkIfExists: true))
+    genomad_db = file(params.genomad_db, checkIfExists: true)
```

Comment:
Using Channel.of was suggested in a previous PR by Vangelis

Reply:
ahh... I rebased this incorrectly, I'll change this and merge